specification/wse/SPEC.md

```diff
@@ -317,26 +319,22 @@ Connection: close
 [emulated websocket frames]
 ```
 
-The downstream response MAY use `Connection: close` or `Transfer-Encoding: chunked` to provide a continuously streaming response
-to the client.
+The downstream response MAY use `Connection: close` or `Transfer-Encoding: chunked` to provide a continuously streaming response to the client.
 
 See [Buffering Proxies](#buffering-proxies) for further client requirements when attaching the downstream.
 
-If the downstream HTTP response transfer is complete, or else complete but does not end in a `RECONNECT` command frame, then the 
-client should consider this as unexpected connection loss for the emulated WebSocket connection.
+If the downstream HTTP response transfer is complete, or else complete but does not end in a `RECONNECT` command frame, then the client should consider this as unexpected connection loss for the emulated WebSocket connection.
```

Comment:
I have removed ", or else complete" so it now reads "If the downstream HTTP response transfer is complete but does not end in a `RECONNECT` command frame, then the client should consider..."